projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-block-editor-assets.php

```diff
@@ -469,6 +472,149 @@ private function setup_block_editor_screen() {
 		}
 	}
 
+	/**
+	 * Removes hooks from problematic plugins that cause errors in this endpoint.
+	 *
+	 * Some plugins conditionally load admin-only code based on is_admin(), which
+	 * returns false in REST API contexts. When these plugins hook into
+	 * enqueue_block_editor_assets without checking the context, they may call
+	 * undefined functions that were never loaded, causing fatal errors.
+	 *
+	 * This method preemptively removes hooks from known problematic plugins before
+	 * the enqueue_block_editor_assets action fires, preventing fatal errors.
+	 */
+	private function remove_problematic_plugin_hooks() {
+		global $wp_filter;
+
+		// List of problematic plugins and their callback patterns
+		$problematic_plugins = array(
+			'wpforms-lite' => array(
+				'enqueue_block_editor_assets',
+			),
+		);
+
```

Comment:
Usually how I would approach this would just be offending callback with a direct remote_action( '' ) call. but this doesn't seem possible with the way the `wpforms-lite` structures its classes as far as I can tell.

Reply:
Agreed. Because of how `wpforms-lite` registers its callback, we need to detect it via reflection and file path.